Pidgin/Parser.Char.cs

```diff
@@ -61,15 +61,14 @@ public static Parser<char, char> AnyCharExcept(params char[] chars)
     /// </summary>
     /// <param name="chars">A sequence of characters that should not be matched.</param>
     /// <returns>A parser which parses and returns a character that does not match one of the specified characters.</returns>
-    public static Parser<char, char> AnyCharExcept(IEnumerable<char> chars)
+    public static Parser<char, char> AnyCharExcept(params char[] chars)
     {
         if (chars == null)
         {
             throw new ArgumentNullException(nameof(chars));
         }
 
-        var cs = chars.ToArray();
-        return Parser<char>.Token(c => Array.IndexOf(cs, c) == -1);
+        return chars.Length == 1 ? Token(c => c != chars[0]) : Token(c => Array.IndexOf(chars, c) == -1);
```

Comment:
If you really wanted to microoptimise this you could copy `chars[0]` into a local first and then just do `Token(c => c != theChar)`

Reply:
According to my benchmark, the performance of using local variables and arrays is almost identical